examples/tinywl/surfacewrapper.cpp

```diff
@@ -76,6 +77,31 @@ SurfaceWrapper::SurfaceWrapper(QmlEngine *qmlEngine, WToplevelSurface *shellSurf
         m_surfaceItem->setFocusPolicy(Qt::NoFocus);
 #endif
     }
+
+    if (type == Type::XWayland) {
+        auto xwaylandSurface = qobject_cast<WXWaylandSurface *>(shellSurface);
+        auto xwaylandSurfaceItem = qobject_cast<WXWaylandSurfaceItem*>(m_surfaceItem);
+
+        xwaylandSurface->safeConnect(&WXWaylandSurface::requestConfigure, this, [this, xwaylandSurfaceItem]() {
+            if (m_xwaylandPositionFromSurface)
+                moveNormalGeometryInOutput(xwaylandSurfaceItem->requestSurfaceWrapperPosition());
+        });
+
+        connect(this, &QQuickItem::xChanged, xwaylandSurface, [this, xwaylandSurfaceItem]() {
+            xwaylandSurfaceItem->setSurfaceWrapperPosition(position(), !m_xwaylandPositionFromSurface);
+        });
+
+        connect(this, &QQuickItem::yChanged, xwaylandSurface, [this, xwaylandSurfaceItem]() {
+            xwaylandSurfaceItem->setSurfaceWrapperPosition(position(), !m_xwaylandPositionFromSurface);
+        });
+
+        // xwaylandSurface->requestConfigureFlags().testAnyFlags(WXWaylandSurface::ConfigureFlag::XCB_CONFIG_WINDOW_POSITION)
+        auto requestPos = xwaylandSurface->requestConfigureGeometry().topLeft();
+        if (!requestPos.isNull()) { // howto check WINDOW_POSITION never configured?
```

Comment:
Use `ConfigureFlag`

Reply:
`requestConfigureFlags` will be override if config twice

Reply:
OK, remvoe the comment